Automattic/wp-calypso
# VideoPress onboarding: Redirect to Home page editor instead of FSE for first video upload

#### Proposed Changes

* When directing the customer to upload their first video, send them to the `Home` page editor, instead of full site editor. This will reduce the number of clicks needed to select and update the video, and also show them a more common interface that they will be using to build pages

cc @evilluendas for a designer opinion on not sending the customer to FSE anymore

| before | after |
|---------|---------|
| <img width="1681" alt="Screenshot 2022-11-16 at 2 30 48 PM" src="https://user-images.githubusercontent.com/4081020/202276227-c4ee0d63-aa68-418e-b2f3-739fdd813394.png"> | <img width="1684" alt="Screenshot 2022-11-16 at 2 28 21 PM" src="https://user-images.githubusercontent.com/4081020/202275709-406c20f1-ef12-44e3-b913-e938900e28ae.png"> |

#### Testing Instructions

~~**NOTE: This may not work on a non-english site since `home` could be translated. If using this page is the approach we want to take with walking the customer through their first video upload then I will continue looking for a language-agnostic solution to load this page**~~



* use Calypso live link and visit `/setup/videopress`
* Go through the flow all the way to the Launchpad
* Click "Upload your first video"
* You should be taken to the `/page` editor for the `Home` page (url will have the page's id)
* Only a single click should be necessary for selecting the video
* Upload a new video and save once upload has completed
* You should see the celebration modal

#### Pre-merge Checklist





Related to # https://github.com/Automattic/greenhouse/issues/1389
